redis.conf

```diff
@@ -1333,10 +1333,10 @@ appendonly no
 # The base name of the append only file.
 # 
 # Redis can persist the data to a set of append-only files, they are divided
-# into two types, one is the the base type, it represents an initial (RDB
-# serialized) snapshot of the data present when the AOF is rewritten. The others
-# are incremental type, they contain the incremental commands since the last AOF
-# rewrite.
+# into two types, one is the the BASE type, it represents an initial (RDB
+# or AOF format) snapshot of the data present when the AOF is rewritten. The 
+# others are INCR type, they contain the incremental commands since the last 
+# AOF rewrite.
```

Comment:
I don't think the documentation should refer to "BASE" and "INCR" as terms or format, but rather use plain English ("base" and "incremental")
```suggestion
# into two types, one is the the base type, it represents an initial (RDB
# or AOF format) snapshot of the data present when the AOF is rewritten. The 
# others are incremental type, they contain the incremental commands since the last 
# AOF rewrite.
```

Reply:
Fix